stacks/aws.py

```diff
@@ -0,0 +1,57 @@
+from .utils import throttling_retry
```

Comment:
aws_helpers or something might be a better name than just aws thinking about it now?